core/src/main/resources/jenkins/model/Jenkins/help-rawBuildsDir.html

```diff
@@ -1,15 +1,21 @@
 <div>
-    Specify where Jenkins would store records of the past builds.
-    This value can include the following variables.
-
+    Specify where Jenkins should store build records. This includes the console
```

Comment:
should implies that it may decide to say "nah screw you" I'm storing it somewhere else.

either this should be "will store" or "by default will store" (as I guess you can change this with some extension point!)